MagneticField/GeomBuilder/plugins/dd4hep/MagGeoBuilder.h

```diff
@@ -0,0 +1,98 @@
+#ifndef MagneticField_GeomBuilder_MagGeoBuilder_H
+#define MagneticField_GeomBuilder_MagGeoBuilder_H
+
+/** \class MagGeoBuilder
+ *  Parse the XML magnetic geometry, build individual volumes and match their
+ *  shared surfaces. Build MagVolume6Faces and organise them in a hierarchical
+ *  structure. Build MagGeometry out of it.
+ *
+ *  \author N. Amapane - INFN Torino
```

Comment:
I deleted these author lines.

Reply:
I'm curious about the rationale about deleting author lines? At least they allow to figure out whom to ask if questions/issues on the code arise. What do we gain in removing this information?

Reply:
> this seems incomplete now

this implied that a major modification made in this PR should have an appropriate attribution

Reply:
> 
> this implied that a major modification made in this PR should have an appropriate attribution

I agree but still I think it is useful to track authors of the code esp. since, besides technicalities, the underlying functional part is still the same. Maybe it's not so relevant here since this is a straightforward header, but it is where actual functionality is implemented, esp. when commit history gets lost because of moving and renaming files. 
It's not just a matter of attribution, it's a matter of keeping tracks who can answer why the code was designed and implemented in a given way, which I think is useful for a code base that was written more than 15 years ago.